x/leveragelp/keeper/begin_blocker.go

```diff
@@ -18,6 +18,7 @@ func (k Keeper) BeginBlocker(ctx sdk.Context) {
 
 	if epochPosition == 0 { // if epoch has passed
 		pools := k.GetAllPools(ctx)
+		// TODO: Optimise fallback system by dividing traversal in multiple blocks
```

Comment:
@amityadav0 how does the fallback system works I thought we could disable it with a feature flag, by the look of the code it looks like it will check for unhealthy positions across all the positions and liquidate or stop loss close

Reply:
we are still discussing it

Reply:
added flag